content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,64 @@
+# Machine Learning Model Registry
+
+A **model registry** is a repository to catalog ML models and their versions.
+Artifacts from your data science projects can be discovered, audited, shared,
+tested, and deployed from here.  
+DVC works on top of Git to enable these capabilities on your existing software
+engineering stack. Other tools from [Iterative](https://iterative.ai/) provide
+complementary features and more usability.
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries is between development and applications. See also [Data Registries]._
+
+[data registries]: /doc/use-cases/data-registries
+
+Model registries give your team key collaborative capabilities:
+
+- Store and present your model artifacts to others, from early-stage
+  [experiment] results to production-ready models.
+- Collect, organize, and track the [versions] of different models effectively;
+  with full data provenance and lineage trails based on [reproducible modeling].
+- Save performance [metrics and plots] or other metadata to evaluate or compare
+  models.
+- An interface to access specific model versions from other systems (e.g. in
+  CI/CD)
+- Centralize ML models from multiple projects into a global repository, if
```

Comment:
ppl won't understand this ...

Reply:
Removed. I suppose the centralization feature is implied in other bullets and explanations. Not really the goal anyway, just a possible byproduct of this pattern